src/js/api/justification.js

```diff
@@ -104,3 +105,70 @@ ripe.Ripe.prototype.getJustificationsByContextP = function(context, options) {
         });
     });
 };
+
+/**
+ * Gets the existing justifications, according to the provided filtering
+ * strategy as normalized values.
+ *
+ * Attempts to resolve context, code and full code and adds those
+ * parameters to the filters provided.
+ *
+ * If the 'other' keyword is contained in the provided `codeFull`, it means
+ * that no RIPE Core justification was used and the custom text provided is
+ * the text that is returned.
+ *
+ * @param {Object} options An object of options to configure the request.
+ * @param {Function} callback Function with the result of the request.
+ * @param {String} other The 'other' prefix code.
+ * @returns {XMLHttpRequest} The XMLHttpRequest instance of the API request.
+ */
+ripe.Ripe.prototype._getJustifications = function(options, callback, other = "other;") {
+    // resolve context, code and full code based on available
+    // information, using smarter filters
+    const params = this._resolveJustificationParams(options);

```

Comment:
early exit when no $ is present

Reply:
After some thinking, I find better if the $ handling remains in the client. Otherwise, will the $ be in the code full, or in the context, when one is given? That does not map well for developers using the RIPE Core API.

In other words, this intuitively seems better (because it is a one-on-one mapping of variables to model fields):
![image](https://user-images.githubusercontent.com/16060539/161557663-92462b37-4188-4d18-940b-204da18bdae2.png)


Than this:
![image](https://user-images.githubusercontent.com/16060539/161557785-5c079a04-7142-4353-b466-2a7f8c84db60.png)